### DISCUSSION TITLE
Schermata di caricamento

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#288] [Idee e proposte]
Sarebbe possibile inserire la schermata di caricamento come in axios ogni volta che si clicca su un tasto?

**Comment by trinko** [comment]
Direi che si potrebbe fare, ma richiede una modifica alla gestione della visualizzazione delle pagine, quindi non so quando sarà possibile inserirla.
La segno come richiesta.
